.openpublishing.redirection.json

```
@@ -35,7 +35,11 @@
         },
         {
             "source_path": "docs/architecture/modernize-with-azure-containers/lift-and-shift-existing-apps-devops/modernize-your-apps-lifecycle-with-ci-cd-pipelines-and-devops-tools-in-the-cloud.md",
-            "redirect_url": "/dotnet/architecture/modernize-with-azure-containers/modernize-existing-apps-to-cloud-optimized/modernize-your-apps-lifecycle-with-ci-cd-pipelines-and-devops-tools-in-the-cloud"
+            "redirect_url": "/dotnet/architecture/modernize-with-azure-containers/modernize-existing-apps-to-cloud-optimized/modernize-lifecycle-ci-cd-pipelines-devops-cloud-tools"
+        },
+        {
+            "source_path": "docs/architecture/modernize-with-azure-containers/modernize-your-apps-lifecycle-with-ci-cd-pipelines-and-devops-tools-in-the-cloud.md",
+            "redirect_url": "/dotnet/architecture/modernize-with-azure-containers/modernize-existing-apps-to-cloud-optimized/modernize-lifecycle-ci-cd-pipelines-devops-cloud-tools"
```

Here's my suggestion given we're renaming it (and I believe life cycle is separate):

```suggestion
            "redirect_url": "/dotnet/architecture/modernize-with-azure-containers/modernize-existing-apps-to-cloud-optimized/life-cycle-ci-cd-pipelines-devops-tools"
```

This would remove duplicate words from the path which are flagged by our SEO scans

---

Nice! updated.